src/forms/Typeahead.jsx

```diff
@@ -14,32 +14,91 @@ const defaultItemToString = itemValue => (typeof itemValue === 'string' ? itemVa
  * @module Typeahead
  */
 class Typeahead extends React.PureComponent {
+	constructor(props) {
+		super(props);
+
+		this.handleSelection = this.handleSelection.bind(this);
+		// this.handleSingleSelection = this.handleSingleSelection.bind(this);
+		// this.handleMultiselectSelect = this.handleMultiselectSelect.bind(this);
+		this.stateReducer = this.stateReducer.bind(this);
+	}
+
+	// handleMultiselectSelect(item) {
+	// 	this.props.onSelect(item, [...this.props.multiSelectValues, item]);
+	// }
+
+	// handleSingleSelection(selectedItem, stateAndHelpers) {
```

Comment:
Do these lines of code still need to be here if commented out?

Reply:
wooops. forgot to delete